src/widget.js

```diff
@@ -102,6 +102,41 @@ export default function (base) {
 		static view_name = "AnyView";
 		static view_module = name;
 		static view_module_version = version;
+
+		/** @param {Parameters<InstanceType<base["DOMWidgetModel"]>["initialize"]>} args */
+		initialize(...args) {
+			super.initialize(...args);
+
+			this.on("change:_css", () => {
+				load_css(this.get("_css"), this.get("_anywidget_id"));
+			})
+
+			this.on("change:_esm", async () => {
```

Comment:
listening for `change` events here instead of custom messages will also make this easier to extend whatever "watcher" we come up with for widgets that derive from the ipywidgets/traitlests based `anywidget.AnyWidget`.